doc/gen-function-list.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
```

Comment:
Changed to pick python from PATH. This solved one of the issue I faced while running.